classic/src/view/panel/layer/Metadata.js

```diff
@@ -12,317 +12,485 @@ Ext.define('MoMo.admin.view.panel.layer.Metadata',{
+                        validator: function(val) {
+                            var startsWith = val.startsWith(' ');
+                            var startsWithErrMsg = 'No whitespaces allowed at the beginning!';
+
+                            var containsTwoWhitespaces = val.indexOf('  ') > -1;
+                            var containsTwoWhitespacesErrMsg = 'Not more than one whitespaces allowed!';
+
+                            if(! startsWith && ! containsTwoWhitespaces) {
+                                return true;
+                            }
+                            else if((startsWith && ! containsTwoWhitespaces) || (!startsWith && containsTwoWhitespaces)) {
+
+                                return startsWithErrMsg || containsTwoWhitespacesErrMsg;
+                            }
+                            else if(!startsWith && containsTwoWhitespaces) {
+                                return containsTwoWhitespacesErrMsg;
+                            }
+                            else {
+                                return startsWithErrMsg +' '+ containsTwoWhitespacesErrMsg;
+                            }
+                        },
                         bind: {
-                            fieldLabel: '{i18n.metadata.deliveryPoint}',
-                            value: '{metadata.organisation.address.deliveryPoint}'
+                            fieldLabel: '{i18n.metadata.name}',
+                            value: '{metadata.organisation.name}'
                         }
                     }, {
                         xtype: 'textfield',
-                        name: 'metadata-organisation-address-postalCode',
-                        allowBlank: false,
+                        name: 'metadata-organisation-website',
+                        vtype: 'url',
                         bind: {
-                            fieldLabel: '{i18n.metadata.postalCode}',
-                            value: '{metadata.organisation.address.postalCode}'
+                            fieldLabel: '{i18n.metadata.website}',
+                            value: '{metadata.organisation.website}'
                         }
                     }, {
+                        xtype: 'fieldset',
+                        name: 'metadata-organisation-address',
+                        width: '100%',
+                        bind: {
+                            title: '{i18n.metadata.address}'
+                        },
+                        defaults: {
+                            width: '100%'
+                        },
+                        items: [{
+                            xtype: 'textfield',
+                            name: 'metadata-organisation-address-deliveryPoint',
+                            allowBlank: false,
+                            msgTarget: 'under',
+                            validator: function(val) {
+                                var startsWith = val.startsWith(' ');
```

Comment:
The validator function has been moved in a new util class MoMo.admin.util.TextfieldValidator. Here Ext.String.startsWith() ist used. See 96e4936